test/functionalTest/testHarness.sh

```diff
@@ -640,6 +666,7 @@ function runTest()
   fileCreation $path $filename
   if [ "$toBeStopped" == "yes" ]
   then
+    runTestStatus="stopped2"
```

Comment:
"stopped2" and not "stopped", as "stopped" has already been used.
I don't even use these values. Just that it is != "OK"